cinema/models.py

```diff
@@ -1,6 +1,11 @@
+import pathlib
+import uuid
+from pathlib import Path
```

Comment:
You have imported the `pathlib` module twice. Once with `import pathlib` and once with `from pathlib import Path`. You only need to import it once. You can remove the `import pathlib` line and replace all uses of `pathlib.Path` with `Path`. This will make your code cleaner.